salt/modules/file.py

```diff
@@ -8,7 +8,7 @@
 # TODO: We should add the capability to do u+r type operations here
```

Comment:
I'm sorry, I don't follow you at all here.

Reply:
The idea internally salt is Unicode/UTF-8.   A template can be UTF-8 on the master, and need to output to Greek language 8bit (non-unicode) on the local minion.

A text file on the minion might be Latin language which needs to be convert correctly into Unicode, so it can be processed within salt to return results to the master.  e.g. `salt latin-box cmd.run rd_text_type=latin cat latin-file`     output to the screen is Unicode/UTF-8 on the master. Very bad example.

Reply:
OK. I understand the use case now, but this is so clearly outside the scope of this PR.